### DISCUSSION TITLE
Do not autorelocate or make you decide between options on port occupied

### DISCUSSION THREAD

**Discussion by miguelgargallo** [coollabsio/coolify#3482] [Feature Requests]
### Description

A place to know what ports we used, and options to choose

![Do not autorelocate or make you decide between options on port occupied](https://github.com/coollabsio/coolify/assets/5947268/de8c2968-bc6d-44b4-afa7-39cd26f69f14)

### Minimal Reproduction (if possible, example repository)

Just deploy on a port used

### Version

v4.0.0-beta.113 Latest

**Comment by peaklabs-dev** [comment]
This is more of a New feature. As a UI to see what ports are in use, and changing the port ranges and IP ranges would be a useful new Feature to add in the future
